osc-server/src/main/java/org/osc/core/broker/rest/client/openstack/openstack4j/Openstack4JNeutron.java

```diff
@@ -298,35 +296,11 @@ public synchronized NetFloatingIP createFloatingIp(String region, String network
         builder.floatingNetworkId(networkId);
         builder.portId(portId);
 
-        NetFloatingIP netFloatingIP;
-        try {
-            netFloatingIP = getOs().networking().floatingip().create(builder.build());
-        } catch (ClientResponseException e) {
-            // Try to remove unused floating IP's assigned to the project
-            if (e.getStatusCode().getCode() == OPENSTACK_CONFLICT_STATUS) {
-                releaseUnusedFloatingIps(region);
-                log.info("Retrying creating floating ip to solve the problem: " + e.getMessage());
-                netFloatingIP = getOs().networking().floatingip().create(builder.build());
-            } else {
-                throw e;
-            }
-        }
-
+        NetFloatingIP netFloatingIP = getOs().networking().floatingip().create(builder.build());
```

Comment:
rethrow the fault in case actionResponse is not a success.

for 404, you can ignore the response but log a warning